samples/test/test.js

```diff
@@ -14,17 +14,19 @@
 
 'use strict';
 
+// eslint-disable-next-line node/no-unpublished-require
```

Comment:
This looks like a bug in `samples/package.json`.  Can you make sure the `files` property is set in that file?  We shouldn't need this exclusion rule.

Reply:
added 
```
"files": [
    "*.js"
  ]
```
Thanks